backends/ebpf/psa/ebpfPsaTable.cpp

```diff
@@ -175,6 +213,17 @@ void EBPFTablePSA::emitTypes(CodeBuilder* builder) {
     // TODO: placeholder for handling PSA-specific types
 }
 
+/**
+ * Remember that order of emitting counters and meters affects future access to BPF maps.
```

Comment:
I find this comment mysterious. "future"? In which way does it affect the access?